native/NativeSampleApps/RestExplorer/src/com/salesforce/samples/restexplorer/RestExplorerApp.java

```diff
@@ -92,15 +94,21 @@ private RestExplorerSDKManager(Context context, Class<? extends Activity> mainAc
 		 * @param context Application context.
 		 * @param mainActivity Activity that should be launched after the login flow.
 		 */
-		public static void initNative(Context context, Class<? extends Activity> mainActivity) {
+		public static void initNative(
+				@NonNull Context context,
+				@NonNull Class<? extends Activity> mainActivity
+		) {
 			if (SalesforceSDKManager.INSTANCE == null) {
 				SalesforceSDKManager.INSTANCE = new RestExplorerSDKManager(context, mainActivity, LoginActivity.class);
 			}
 			initInternal(context);
 		}
 
+		@NonNull
 		@Override
-		public LinkedHashMap<String, DevActionHandler> getDevActions(Activity frontActivity) {
+		public Map<String, DevActionHandler> getDevActions(
```

Comment:
Here's another Collections implementation to interface update downstream of Kotlin conversion.